nested_open_struct.gemspec

```diff
@@ -6,7 +6,10 @@ Gem::Specification.new do |s|
   s.homepage    = "http://github.com/porras/nested_open_struct"
   s.summary     = "Thin wrapper around OpenStruct to make structs 'nested'"
   s.description = "Thin wrapper around OpenStruct to make structs 'nested'"
- 
+
   s.files        = Dir.glob("lib/**/*") + %w(LICENSE README.md)
   s.require_path = 'lib'
-end
\ No newline at end of file
+
+  s.add_development_dependency 'minitest', '~> 5.8'
+  s.add_development_dependency 'rake',     '~> 10.4'
```

Comment:
> Also yes, :+1: no Gemfile.lock for libraries, Gemfile.lock is for applications :)

Oh, I forgot that part, I'll remove it in `master`, because I can :sunglasses: